docs/api-docs/catalog/products-overview.md

```diff
@@ -744,18 +740,18 @@ An adjuster can be added to a modifier option to change things such as increasin
 
 <!-- theme:  -->
 
-### Modifiers that support Adjusters
-> Swatch, radio buttons, rectangle list, drop-down, product list, product list with images
+### Modifiers that support adjusters
+> swatch, radio buttons, rectangle list, drop-down, product list, and product list with images
 
 </div>
 </div>
 </div>
 
 ### Add a modifier with price adjuster to an existing product
 
-The following example shows how to add a modifier, a checkbox with a price adjuster, that will increase the product's price by five dollars.
+The following example shows how to add a modifier and a checkbox with a price adjuster to increase the product's price by five dollars.
 
-Creating a checkbox with an adjuster requires two separate calls: one to create the checkbox and a second to add the adjuster. Adjusters are defined within the `option_values` array, but `option_values` are not allowed in the request to create a checkbox modifier because creating a checkbox automatically generates two mandatory option values: a Yes and a No. Once the checkbox has been created along with its option values, you can update the modifier to add an adjuster.
+Creating a checkbox with an adjuster requires two separate calls: one to create the checkbox and a second to add the adjuster. You can define adjusters within the `option_values` array, but `option_values` are not allowed in the request to create a checkbox modifier because creating a checkbox automatically generates two mandatory option values: a Yes and a No. Once you have created the checkbox along with its option values, you can update the modifier to add an adjuster.

```

Comment:
```suggestion
Creating a checkbox with an adjuster requires two separate calls: one to create the checkbox and another one to add the adjuster. You can define adjusters within the `option_values` array, but `option_values` are not allowed in the request to create a checkbox modifier because creating a checkbox automatically generates two mandatory option values: a Yes and a No. Once you have created the checkbox along with its option values, you can update the modifier to add an adjuster.

```